doc/airline.txt

```diff
@@ -1662,7 +1662,7 @@ vim-windowswap <https://github.com/wesQ3/vim-windowswap>
   " The default value matches filetypes typically used for documentation
   " such as markdown and help files. Default is:
   let g:airline#extensions#wordcount#filetypes =
-    \ ['asciidoc', 'help', 'mail', 'markdown', 'nroff', 'org', 'plaintex', 'rst', 'tex', 'text'])
+    \ ['asciidoc', 'help', 'mail', 'markdown', 'rmd' 'nroff', 'org', 'plaintex', 'rst', 'tex', 'text'])
```

Comment:
missing `,` between `'rmd'` and `'nroff'`

Reply:
Fixed in the latest commit.